lib/response.js

```diff
@@ -123,10 +123,10 @@ function newNote (req, res, next) {
   }
   models.Note.create({
     ownerId: owner,
-    alias: req.alias ? req.alias : null,
+    alias: req.alias ? req.alias : (config.allowFreeURL ? (req.params.alias ? req.params.alias : null) : null),
     content: body
   }).then(function (note) {
-    return res.redirect(config.serverURL + '/' + models.Note.encodeNoteId(note.id))
+    return res.redirect(config.serverURL + '/' + (note.alias ? note.alias : models.Note.encodeNoteId(note.id)))
```

Comment:
I think we can still redirect to the noteId even when the note alias exists. This otherwise just makes everything more complicated.

Reply:
In which matter does it make everything more complicated? I would see the redirect as a form of confirmation that the import succeeded. When the user is redirected to the noteId instead of the alias, he might assume that the alias was not properly set.

Reply:
Mhm, good point, @ccoenen what's your input on this?

Reply:
The alias'd version takes precedence in other places, so we should do so here as well.

(the following lines are just a few examples I found quickly)

https://github.com/codimd/server/blob/529075fd67ad0d41ba0b25a7de69817dd4cd76fb/lib/response.js#L185

https://github.com/codimd/server/blob/529075fd67ad0d41ba0b25a7de69817dd4cd76fb/lib/response.js#L202

https://github.com/codimd/server/blob/529075fd67ad0d41ba0b25a7de69817dd4cd76fb/lib/response.js#L249

https://github.com/codimd/server/blob/529075fd67ad0d41ba0b25a7de69817dd4cd76fb/lib/response.js#L253